newrelic_rpm.gemspec

```diff
@@ -47,16 +47,17 @@ https://github.com/newrelic/newrelic-ruby-agent/
   s.homepage = "https://github.com/newrelic/rpm"
   s.require_paths = ["lib"]
   s.summary = "New Relic Ruby Agent"
+  s.add_development_dependency 'bundler'
   s.add_development_dependency 'feedjira', '3.2.1' unless ENV['CI'] || RUBY_VERSION < '2.5' # for Gabby
-  s.add_development_dependency 'rake', '12.3.3'
+  s.add_development_dependency 'httparty' unless ENV['CI'] # for perf tests and Gabby
   s.add_development_dependency 'minitest', '5.3.3'
   s.add_development_dependency 'minitest-stub-const', '0.6'
   s.add_development_dependency 'mocha', '~> 1.14.0'
-  s.add_development_dependency 'yard'
   s.add_development_dependency 'pry' unless ENV['CI']
-  s.add_development_dependency 'bundler'
+  s.add_development_dependency 'rake', '12.3.3'
   s.add_development_dependency 'rubocop'
   s.add_development_dependency 'rubocop-performance'
   s.add_development_dependency 'simplecov' if RUBY_VERSION >= '2.7.0'
-  s.add_development_dependency 'httparty' unless ENV['CI'] # for perf tests and Gabby
+  s.add_development_dependency 'thor'
+  s.add_development_dependency 'yard'
```

Comment:
As our development dependencies grow, I thought alphabetizing them might be handy

Reply:
Thank you